lib/pages/chat/chat.dart

```diff
@@ -136,13 +147,23 @@ class ChatController extends State<Chat>
       );
 
   final AutoScrollController scrollController = AutoScrollController();
+  final AutoScrollController pinnedMessageScrollController =
+      AutoScrollController();
   final KeyboardVisibilityController keyboardVisibilityController =
       KeyboardVisibilityController();
   final ValueNotifier<String?> focusHover = ValueNotifier(null);
   final ValueNotifier<bool> openingPopupMenu = ValueNotifier(false);
   final ValueNotifier<bool> draggingNotifier = ValueNotifier(false);
   final ValueNotifier<bool> showScrollDownButtonNotifier = ValueNotifier(false);
   final ValueNotifier<bool> showEmojiPickerNotifier = ValueNotifier(false);
+  final ValueNotifier<String?> currentPinnedMessageIdNotifier =
+      ValueNotifier(null);
+  final ValueNotifier<Event?> isCurrentPinnedEventNotifier =
```

Comment:
isCurrentPinnedEventNotifier and currentPinnedMessageIdNotifier make confusion

Reply:
done